sites/all/themes/wille/templates/system/html.tpl.php

```diff
@@ -59,5 +59,6 @@
   <?php print $page; ?>
   <?php print $scripts; ?>
   <?php print $page_bottom; ?>
+  <script src="/sites/all/themes/wille/overlay/overlay.js"></script>
```

Comment:
Use https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_get_path/7.x to get the theme path.